lib/spandx/core/data_file.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
```

Comment:
@mokhan 
We should definitely write some comments on top of this file `Spandx::Core::IndexFile` .  Just to tell what they are doing